SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -178,6 +178,17 @@ USpatialLatencyTracer* USpatialLatencyTracer::GetTracer(UObject* WorldContextObj
 	return nullptr;
 }
 
+FString USpatialLatencyTracer::GetTraceMetadata(UObject* WorldContextObject)
+{
+#ifdef TRACE_LIB_ACTIVE
+	if (USpatialLatencyTracer* Tracer = GetTracer(WorldContextObject))
+	{
+		return Tracer->TraceMetadata;
```

Comment:
As this is set by the project explicitly, do we need an accessor?

Reply:
Yes, we need this in order to get the TraceMetadata name from the simulated players and pass it to the server workers. As they are in different projects and the nfr framework runner uses the simulated player deployment name as the run id, this is the most straight forward way to get the run id. We use this function from a blueprint to set the server worker's run id to match the run id of the simulated players.